django/core/management/commands/makemigrations.py

```diff
@@ -346,7 +346,9 @@ def write_migration_files(self, changes, update_previous_migration_paths=None):
                     migration_string = self.get_relative_path(writer.path)
                     self.log("  %s\n" % self.style.MIGRATE_LABEL(migration_string))
                     for operation in migration.operations:
-                        self.log("    - %s" % operation.describe())
+                        self.log(
+                            " %s" % operation.formatted_description(style=self.style)
```

Comment:
Is there a reason to remove the indentation?

Reply:
My bad. Thnx for spotting!